src/radical/saga/adaptors/slurm/slurm_job.py

```diff
@@ -674,7 +674,6 @@ def _job_run(self, job_obj):
 
                 assert(n_nodes), 'need unique number of cores per node'
                 script += "#SBATCH -N %d\n" % n_nodes
-                script += "#SBATCH -n %d\n" % n_procs
```

Comment:
I've revert it back, since this section is with option `-N` only, but previous section has both options `-N` and `-n`